src/nl/surf/eduhub_rio_mapper/http_utils.clj

```diff
@@ -6,6 +6,13 @@
 
 ;; middleware to add to the http client
 
+(defn ^:dynamic recorder [handler request]
```

Comment:
Maak hier dan `*vcr*` van denk ik en een docstring erbij waar het voor is en waarom dynamisc